lightyear/src/connection/netcode/server.rs

```diff
@@ -898,7 +913,7 @@ impl<Ctx> NetcodeServer<Ctx> {
     pub fn send_all(&mut self, buf: &[u8], io: &mut Io) -> Result<()> {
         for id in self.conn_cache.ids() {
             match self.send(buf, id, io) {
-                Ok(_) | Err(Error::ClientNotConnected) | Err(Error::ClientNotFound) => continue,
+                Ok(_) | Err(Error::ClientNotConnected) | Err(Error::ClientNotFound(_)) => continue,
```

Comment:
Maybe i missed it, but in case of an error don't we want to call `self.handle_client_error`?

This method doesn't seem used at all, anyway

Reply:
Ah it's because for `send` there is no buffering so we can just return the error directly?

Reply:
that's right